src/libs/actions/PersonalDetails.js

```diff
@@ -215,14 +215,27 @@ function mergeLocalPersonalDetails(details) {
  * @param {boolean} shouldGrowl
  */
 function setPersonalDetails(details, shouldGrowl) {
-    API.PersonalDetails_Update({details: JSON.stringify(details)});
-    if (details.timezone) {
-        NameValuePair.set(CONST.NVP.TIMEZONE, details.timezone);
-    }
-    mergeLocalPersonalDetails(details);
-    if (shouldGrowl) {
-        Growl.show(translateLocal('profilePage.growlMessageOnSave'), CONST.GROWL.SUCCESS, 3000);
-    }
+    API.PersonalDetails_Update({details: JSON.stringify(details)})
+        .then((response) => {
+            if (response.jsonCode === 200) {
+                if (details.timezone) {
+                    NameValuePair.set(CONST.NVP.TIMEZONE, details.timezone);
+                }
+                mergeLocalPersonalDetails(details);
+            }
+
+            if (shouldGrowl) {
+                if (response.jsonCode === 400) {
+                    Growl.error(translateLocal('profilePage.error.messageOn400'), 3000);
+                } else if (response.jsonCode === 401) {
+                    Growl.error(translateLocal('profilePage.error.messageOn401'), 3000);
```

Comment:
What do you think about taking these outside of the `if (shouldGrowl)` condition? I think it would be nice to show these errors no matter what. Thoughts?

Reply:
Sure, we can do that but initially I thought that since `shouldGrowl` must be `true` whenever a feedback is needed, it should not be a problem. 

> I think it would be nice to show these errors no matter what.

I think this code should satisfy this requirement. 
Let me know if I should update the PR.

```js
            if (shouldGrowl && response.jsonCode === 200) {
                Growl.show(translateLocal('profilePage.growlMessageOnSave'), CONST.GROWL.SUCCESS, 3000);
            } else if (response.jsonCode === 400) {
                Growl.error(translateLocal('profilePage.error.messageOn400'), 3000);
            } else if (response.jsonCode === 401) {
                Growl.error(translateLocal('profilePage.error.messageOn401'), 3000);
            }
```

Reply:
Sorry for the delay - yeah I see what you mean, but I believe `shouldGrowl`'s original purpose was for showing the success growl or not, and to me it makes sense to make sure error growls are shown any time there's an error.

Reply:
So I like the code update you recommended, please add when you have a chance!

Reply:
Updated the code.